### PR TITLE
Improve layout of buttons for two factor auth

### LINKED ISSUE #4298: Fix two factor button layout
As you can see in https://github.com/nextcloud/server/pull/2618#issuecomment-284352757

The buttons look kind of ugly now. They need to be styles properly.
@nextcloud/designers work your magic please!

### PR DESCRIPTION
**Pull request by MorrisJobke**
* if nobody wants to fix it 😝 
* fixes #4298

Before:

bildschirmfoto 2017-05-14 um 22 13 07
bildschirmfoto 2017-05-14 um 22 12 46

After:

bildschirmfoto 2017-05-14 um 22 16 42
bildschirmfoto 2017-05-14 um 22 14 33

### CODE REVIEW

**Comment by ChristophWurst** [comment]
For some reason the logout button looks different depending on whether backup codes are active. If they are, the button takes up 100% of the box (like on your screenshot). If it's inactive, it's narrower:

bildschirmfoto von 2017-05-15 07-37-02

File: core/css/guest.css [hunk 1]
```diff
@@ -336,9 +336,18 @@ form .warning input[type='checkbox']+label {
 }
 .two-factor-link {
 	display: inline-block;
-	padding: 12px;
 	color: rgba(255, 255, 255, 0.75);
 }
+.two-factor-link a.button {
```
Comment (jancborchardt) [inline core/css/guest.css]: This doesn’t need to be that specific, can be simply `.two-factor-link .button` (without the a)

**Review by jancborchardt — COMMENTED** [review]
Looks good except my detail comment and what @ChristophWurst said! :)

**Comment by MorrisJobke** [comment]
Fixed all comments.

**Review by ChristophWurst — APPROVED** [review]
Tested and works! Looks a lot better now, thanks :)

**Review by rullzer — APPROVED** [review]
:balloon:

**Pull request merged**